### PR TITLE
refactor: migrate to Node.js native test runner

### PR DESCRIPTION
**Pull request by fraxken**

### CODE REVIEW

File: test/index.js
Comment (PierreDemailly) [inline test/index.js:1]: We need this even with glob pattern ?

File: test/obfuscated.spec.js [hunk 1]
```diff
@@ -12,84 +11,87 @@ import { getWarningKind } from "./utils/index.js";
...
-  tape.deepEqual(getWarningKind(warnings), ["obfuscated-code"].sort());
-  tape.strictEqual(warnings[0].value, "jsfuck");
-  tape.end();
+  assert.strictEqual(warnings.length, 1);
+  assert.deepEqual(getWarningKind(warnings), ["obfuscated-code"].sort());
+  assert.strictEqual(warnings[0].value, "jsfuck");
 });
 
-// test("should detect 'morse' obfuscation", (tape) => {
+// test("should detect 'morse' obfuscation", () => {
```
Comment (PierreDemailly) [inline test/obfuscated.spec.js:23]: Should we delete or uncomment this test ?

File: test/index.js
Comment (fraxken) [reply test/index.js:1]: Yes because it will try to execute other files (like fixtures) that are not tests

File: test/obfuscated.spec.js (hunk 1 above)
Comment (fraxken) [reply test/obfuscated.spec.js:23]: https://github.com/NodeSecure/js-x-ray/issues/144 (out the scope of this PR)

**Review by PierreDemailly — APPROVED** [review]
LGTM
edit: % tape conflict

**Pull request merged**